examples/shapes_and_collections/fancybox_demo.py

```diff
@@ -15,17 +17,16 @@
 # First we'll show some sample boxes with fancybox.
 
 styles = mpatch.BoxStyle.get_styles()
-spacing = 1.2
-
-figheight = (spacing * len(styles) + .5)
-fig = plt.figure(figsize=(4 / 1.5, figheight / 1.5))
-fontsize = 0.3 * 72
-
-for i, stylename in enumerate(sorted(styles)):
-    fig.text(0.5, (spacing * (len(styles) - i) - 0.5) / figheight, stylename,
-             ha="center",
-             size=fontsize,
-             bbox=dict(boxstyle=stylename, fc="w", ec="k"))
+ncol = 2
+nrow = (len(styles) + 1) // ncol
+axs = plt.figure(figsize=(3 * ncol, nrow)).subplots(nrow, ncol)
+for ax in axs.flat:
+    ax.set_axis_off()
+for ax, (stylename, stylecls) in zip(axs.T.flat, styles.items()):
+    ax.text(.5, .5,
+            f"{stylename}\n{inspect.signature(stylecls)}",
+            transform=ax.transAxes, ha="center", va="center",
+            bbox=dict(boxstyle=stylename, fc="w", ec="k"))
```

Comment:
anyway to make the boxes slightly bigger? especially for the sawtooth, the styling gets a little lost in the smaller size.

Reply:
Actually these *are* the default params, and I'd rather keep using the defaults for the reference docs?

Reply:
Hmm, makes sense, but ugh so hard to differentiate :/  is it cause size is determined by pad? And then hmm, could changing the color help? maybe an inside gray to make the border pop?

Reply:
It's proportional to the fontsize by default.  I'll let you judge whether gray is better...
![test](https://user-images.githubusercontent.com/1322974/123683796-f08fc480-d84c-11eb-866c-3a518e44e3d1.png)